Casks/c/codelite.rb

```diff
@@ -1,25 +1,57 @@
 cask "codelite" do
-  arch arm: "arm64", intel: "x86_64"
+  on_arm do
+    on_monterey do
```

Comment:
```suggestion
    on_monterey :or_older do
```

Our usual pattern is to set the first block to `:or_older` and, if needed, specify a lower bound using a global `depends_on` (e.g., `depends_on macos: ">= :monterey"`).